themes/digital.gov/src/js/common/in-page-navigation.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener("DOMContentLoaded", (event) => {
```

Comment:
Event doesn't seem to be used. Can we remove?

```suggestion
document.addEventListener("DOMContentLoaded", () => {
```

Reply:
Good catch, this is removed.